seed/views/properties.py

```diff
@@ -202,7 +202,7 @@ class PropertyViewViewSet(SEEDOrgModelViewSet):
     queryset = PropertyView.objects.all()
 
 
-class PropertyViewSet(GenericViewSet, ProfileIdMixin):
+class PropertyViewSet(ViewSet, ProfileIdMixin):
```

Comment:
Just curious, was swagger complaining due to unimplemented methods?

Reply:
Yep - it was looking for either the `get_queryset` method or the `queryset` attribute.